app/controllers/application_controller.rb

```diff
@@ -93,6 +93,14 @@ def filter_params
     end
   end
 
+  def oidc
+    @oidc ||= OidcClient.new(
+      ENV["ACCOUNT_MANAGER_URL"],
```

Comment:
Could we get this out of Plek instead?
```suggestion
      Plek.find("account-manager"),
```